Formula/openj9.rb

```diff
@@ -0,0 +1,92 @@
+class Openj9 < Formula
+  desc "High performance, scalable, Java virtual machine"
+  homepage "https://www.eclipse.org/openj9/"
+  url "https://github.com/eclipse/openj9.git",
+    branch:   "v0.25.0-release",
+    tag:      "openj9-0.25.0",
+    revision: "022d65424fab41e054ae9834822414cc5334f5bc"
+  license any_of: [
+    "EPL-2.0",
+    "Apache-2.0",
+    "GPL-2.0-only" => { with: "Classpath-exception-2.0" },
+  ]
+
+  livecheck do
+    url :stable
+    regex(/^openj9-(\d+(?:\.\d+)+)$/i)
+  end
+
+  keg_only :shadowed_by_macos
+
+  depends_on "autoconf" => :build
+  depends_on "bash" => :build
+  depends_on "cmake" => :build
+  depends_on "nasm" => :build
+  depends_on "ninja" => :build
+  depends_on "openjdk" => :build
+  depends_on "pkg-config" => :build
+  depends_on xcode: :build
+
+  depends_on "freetype"
+  depends_on "openssl@1.1"
+
+  resource "openj9-openjdk-jdk16" do
+    url "https://github.com/ibmruntimes/openj9-openjdk-jdk16.git",
+    branch:   "v0.25.0-release",
+    revision: "0c11227a21ace48b7c978e29a347822912a6df9c"
+  end
+
+  def install
+    resource("openj9-openjdk-jdk16").stage buildpath/"openj9-openjdk-jdk16"
+    java_options = ENV.delete("_JAVA_OPTIONS")
+    # shims cc builds a broken `omr_ddrgen`
+    ENV.prepend "PATH", "/usr/bin:"
```

Comment:
Why is this and has it been reported upstream?

Reply:
The reason why `omr_ddrgen` is broken when use cc from shims is still unclear 😕.

Reply:
Not sure there's much point in reporting that our shims don't work. The most likely response is "don't use your shims then" -- it's what I would say.

We should, however, understand why they don't work so `superenv` can be fixed.

Reply:
Now resolved by disable the feature `ddr`.